inc/Engine/Optimization/RUCSS/Warmup/Subscriber.php

```diff
@@ -40,7 +49,8 @@ public function __construct( Options_Data $options, ResourceFetcher $resource_fe
 	 */
 	public static function get_subscribed_events() : array {
 		return [
-			'rocket_buffer' => [ 'collect_resources', 1 ],
+			'rocket_buffer' => [ 'collect_resources', 11 ],
+			'update_option_' . rocket_get_constant( 'WP_ROCKET_SLUG' ) => [ 'start_scanner', 15, 2 ],
```

Comment:
formatting issue related to aligning this `=>`

Reply:
PHPCS doesn't agree, it shows an error if I change it